packages/block-editor/src/components/block-mobile-floating-toolbar/index.native.js

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
This is a mobile-specific component. Please move it to the `packages/components/mobile/`

Reply:
moved